CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 v0.13.3 - Wed, 29 Apr 2015 01:47:08 GMT
```

Comment:
Looks like we never got comments for `v0.13.4`; let's not make that mistake with `v0.13.5`